### PR TITLE
Add unittest for security/pkg/server/grpc

### PR DESCRIPTION
**Pull request by diemtvu**
https://github.com/istio/istio/issues/4008

### CODE REVIEW

File: security/pkg/registry/registry_test.go.orig [hunk 1]
```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 Istio Authors
```
Comment (myidpt) [inline security/pkg/registry/registry_test.go.orig]: Remove this file?

Comment (wattli) [reply security/pkg/registry/registry_test.go.orig]: +1

Comment (diemtvu) [reply security/pkg/registry/registry_test.go.orig]: whoops :)

**Comment by wattli** [comment]
i think you need to rebase and push latest upstream to pass the circleci test.

**Pull request merged**